wp-api.js

```diff
@@ -224,7 +224,7 @@
 		{
 			idAttribute: 'id',
 
-			urlRoot: WP_API_Settings.root + '/users',
+			urlRoot: WP_API_Settings.root + 'wp/v2/users',
```

Comment:
Question @rmccue: If the URL always starts with WP_API_Settings.root + 'wp/v2/', why isn't 'wp/v2/' part of WP_API_Settings.root?

If that doesn't work, how about a 'versionPrefix' variable we can easily update?